docs/tutorials/pytorch-mnist-tutorial.txt

```diff
@@ -118,19 +105,13 @@ The ``build_model`` method returns a ``torch.Module`` or ``torch.Sequential`` ob
             nn.Dropout2d(self.context.get_hparam("dropout2")),
             nn.Linear(128, 10),
             nn.LogSoftmax(),
-        )
-
-        reset_parameters(model)
-        return model
-
-Defining the Optimizer
-""""""""""""""""""""""
-The ``optimizer`` method returns a ``torch.optim`` object. The MNIST model code uses ``torch.optim.Adadelta`` and we can continue to use that in our implementation of ``optimizer``.
+        ))
+        reset_parameters(self.model)
```

Comment:
It actually calls `.reset_parameters()` on each item of `self.model.named_modules()`. So you basically need to write a loop to do that. I would leave it to https://determinedai.atlassian.net/browse/DET-3578

Reply:
Talked over zoom, we'll leave this for a separate PR, but it would be good to fix this before the major api update goes out (that is, in this next release).